### ISSUE TITLE
Create Model for Questions and Answers

### ISSUE THREAD

**Issue by jshear95** [sklei2/DeathClock#36]

**Comment by jshear95** [comment]
This also includes displaying them in the web app.

**Comment by sklei2** [comment]
do you happen to have a branch for this (where you'd make a pull request)? or is it in master now?

**Comment by jshear95** [comment]
I do not have a branch for this. I was unsure of how to make a pull request so I figured we could sort it out tomorrow quickly. Do you want me to make a branch?

**Comment by sklei2** [comment]
Yes Please! if you make a branch then push that branch up to the remote repository. then in github you can easily make a pull request on a branch. Just make sure to pull in master and handle those merges.

**Comment by jshear95** [comment]
I created the questions branch, but when I try to push it, I get this:
Joshuas-MacBook-Pro:DeathClock joshuashearshear$ git push origin questions
error: src refspec questions does not match any.
error: failed to push some refs to 'https://github.com/sklei2/DeathClock.git'

Any ideas?

**Comment by sklei2** [comment]
try
`git push --set-upstream origin questions`

I think that is because your local branch isn't tracking your remote. This makes it so it does and will create a remote branch for you

**Comment by jshear95** [comment]
Same error :/

**Comment by jshear95** [comment]
It says I am on branch 'Q&A' which doesn't exist.

**Comment by jshear95** [comment]
Ok, got it pushed to Q&A apparently creating a branch the command line doesn't create it until the first push. I will delete the questions branch and make a pull request.
